src/store/metadata/oxigraph/store.py

```diff
@@ -258,7 +259,40 @@ def get_publishers(self) -> Optional[Dict]:  # pragma: no cover
         """
         Gets all publishers
         """
-        raise NotImplementedError
+
+        # Specify the named graph from which we are fetching data
+        graph = self.db
+        
+        # Use the construct wrappers to pull the raw RDF triples
+        # (as one rdflib.Graph() for each function) and add them
+        # together to create a sinlge Graph of the
+        # data we need.
+        result: Graph = (
+            construct_publishers(graph)
+        )
+
+        # Serialize the graph into jsonld
+        data = json.loads(result.serialize(format="json-ld"))
+
+        # Use a context file to shape our jsonld, removing long form references
+        data = jsonld.flatten(
+            data, {"@context": constants.CONTEXT, "@type": "hydra:Collection"}
+        )
+
+        # TODO Fix context weirdness - at the moment, the flatten() method is changing @type to `versions_url`
+        # also adds dcat: to the publishers
+        data["@graph"][0]["@type"] = "hydra:Collection"
+        data["@graph"][0]["publishers"] = data["@graph"][0].pop("dcat:publisher")
+
+        for idx, publisher in enumerate(data["@graph"][0]["publishers"]):
+            publisher_id = publisher["@id"].split("/")[-1]
+            data["@graph"][0]["publishers"][idx] = self.get_publisher(publisher_id)
+        
+        # TODO Update @context so it's not hardcoded
+        data["@graph"][0]["@context"] = "https://staging.idpd.uk/ns#"
+        result = data["@graph"][0]
```

Comment:
`@context` is still missing so the test is failing on validation

Reply:
Weird, this wasn't here when i added the comment

Reply:
Yeah it is odd, I don't see why this issue remains but for now it solves it